### PR TITLE
feat(steps): create progress step component

### PR DESCRIPTION
**Pull request by fabricionaweb**
## desktop
image

## mobile
image

### CODE REVIEW

File: src/css/garden.css [hunk 1]
```diff
@@ -56,5 +61,6 @@
 @import "molecules/feedback";
 @import "molecules/collapse";
 @import "molecules/tooltip";
+@import "molecules/steps-stage";
```
Comment (leandrogrillo) [inline src/css/garden.css]: in this case, it's better to keep the same structure as in `atoms`.

`molecules/steps/stage`

File: docs/components/steps.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+</div>
+
+```html
+  <ul class="steps-container">
+    <li class="steps-stage active">
+      <div class="steps-bullet">
+        1
+      </div>
+      <div class="steps-label">
+        Pedido recebido <br>
```
Comment (willamesoares) [inline docs/components/steps.md]: Por que usar português aqui? Seria bom deixar a documentação inteira, inclusive exemplos, toda em inglês.

Comment (fabricionaweb) [inline docs/components/steps.md]: Good point!! My bad

File: src/css/garden.css (hunk 1 above)
Comment (fabricionaweb) [inline src/css/garden.css]: We do not have other components in directories, so I did not feel safe to start

Comment (fabricionaweb) [inline src/css/garden.css]: But i can rename to `steps` to normalize

File: docs/components/steps.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+# Steps
+<div class="lead">
+  Steppers convey progress through numbered steps.
+</div>
+
+## Usage
+
+To get the default style for this component you just have to add the `.steps-container` class to a `<ul>` tag and `.steps-stage` to an `<li>` children.
+
+For the active steppers use `.active` and for the current step use `.current`.
```
Comment (willamesoares) [inline docs/components/steps.md]: what about using `steps` instead of `steppers` here?

**Pull request merged**